discord/guild.py

```diff
@@ -2923,9 +2924,13 @@ async def ban(
         -----------
         user: :class:`abc.Snowflake`
             The user to ban from their guild.
-        delete_message_days: :class:`int`
-            The number of days worth of messages to delete from the user
-            in the guild. The minimum is 0 and the maximum is 7.
+        delete_message_seconds: Optional[:class:`int`]
+            The number of seconds worth of messages to delete from
+            the user in the guild. The minimum is 0 and the maximum
+            is 604800 (i.e. 7 days). The default is 0.
+        delete_message_days: Optional[:class:`int`]
```

Comment:
IMO, we should mark this as deprecated but send it as normal (without converting to seconds) in the payload. When discord removes support, we remove the option.

Reply:
By "remove support" do you mean when the docs pr is merged? when v9 is deprecated?

Reply:
when discord announces it